File: septunneuf-serveur/septunneuf-monsters/src/main/java/fr/ebiron/septunneuf/monsters/model/DatabaseSequence.java
package fr.ebiron.septunneuf.monsters.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "database_sequences")
public class DatabaseSequence {
    @Id
    private String id;
    private Long seq;

    public Long getSeq() {
        return seq;
    }

    public DatabaseSequence() {
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public void setSeq(Long seq) {
        this.seq = seq;
    }
}

File: septunneuf-serveur/septunneuf-eggs/src/main/kotlin/fr/ebiron/septunneuf/eggs/models/DatabaseSequence.kt
package fr.ebiron.septunneuf.eggs.models

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "database_sequences")
data class DatabaseSequence(
    @Id
    val id: String,
    val seq: Long
)
